cypress.config.js

```diff
@@ -1,8 +1,11 @@
-const { defineConfig } = require('cypress')
+const { defineConfig } = require("cypress");
 
 module.exports = defineConfig({
-  e2e: {
-    setupNodeEvents(on, config) {
-    },
+  e2e: {   
+           
+        baseUrl: "http://localhost:1667",
```

Comment:
Next time test the app you are asked for, please (https://react-redux.realworld.io)

Reply:
Ahhhhh..... i even didn't notice it.... i will also do the correct one . thanks